macros/utils/cross_db_utils/edr_day_of_week_expression.sql

```diff
@@ -0,0 +1,34 @@
+{% macro edr_day_of_week_expression(date_expr) %}
```

Comment:
Is the behavior consistent between all DWHs? 
Do they all return the exact same when running this on the same date?

Reply:
> Is the behavior consistent between all DWHs?
It's not - it's not even consistent in specific DWH version, since it's based on some account level definitions of the time the week starts etc. 
We only require for the day_of_week seasonality to have the same retval here for the same days of the week, which is the case as it is.

I think going for 100% consistency in the DWHs we support would be a lot of effort in dates handling internals and not sure what value it would bring.

Reply:
Using the name of the day will give a good 99.9% consistency